unimport/utils.py

```diff
@@ -31,6 +32,7 @@ def get_source(import_name: str) -> Optional[str]:
     return None
 
 
+@functools.lru_cache(maxsize=None)
```

Comment:
If maxsize is set to None, the LRU feature is disabled and the cache can grow without bound.

```suggestion
@functools.lru_cache()
```